src/seth/README.md

```diff
@@ -63,6 +63,7 @@ Contents
       * [`seth --to-uint256`]
       * [`seth --to-bytes32`]
       * [`seth --to-address`]
+      * [`seth gas-price`]
```

Comment:
Can you please add it in alphabetical order?

Reply:
Indeed